src/openapi.yaml

```diff
@@ -18,6 +18,23 @@ paths:
 
   # TODO: add paths here
 
+  # user endpoints
```

Comment:
Please put all endpoints concerning users under `/users`.

Lets say we make an API request like `https://peerreq.com/categories/...`
Is this a request about categories or a user with the `id` _categories_?

You normally do not want a parameter at the path root.

Reply:
Please include the path parameters in the directory structure as well.